DP_3_Problem_2.java

```diff
@@ -0,0 +1,33 @@
+//Time Complexity : O(n)
+//Space Complexity: O(1)
+//Wont run for colums > 3
```

Comment:
Please try solving based on my other comment. Let me know if you still face the same issue